boards/arm/nrf5340_dk_nrf5340/doc/index.rst

```diff
@@ -0,0 +1,259 @@
+.. _nrf5340_dk_nrf5340:
+
+nRF5340-DK-NRF5340
+##################
+
+Overview
+********
+
+The nRF5340 DK NRF5340 is a single-board development kit for evaluation
+and development on the Nordic nRF5340 System-on-Chip (SoC).
+
+nRF5340 is a dual-core SoC based on the Arm® Cortex®-M33 architecture, with
```

Comment:
Some formatting fixes:
```
The nRF5340 is a dual-core SoC based on the Arm® Cortex®-M33 architecture, with:

* a full-featured ARM Cortex-M33F core with DSP instructions, FPU, and
  ARMv8-M Security Extension, running at up to 128 MHz, referred to as
  the **Application MCU**
* a secondary ARM Cortex-M33 core, with a reduced feature set, running at
  a fixed 64 MHz, referred to as the **Network MCU**.
```

Reply:
@dbkinder thanks for the review, suggestion applied - please do a quick pass :)